src/newton.jl

```diff
@@ -97,7 +97,7 @@ function newton_{T}(df::AbstractDifferentiableMultivariateFunction,
     end
     function fgo!(xlin::Vector, storage::Vector)
         go!(xlin, storage)
-        return(dot(fvec, fvec) / 2)
+        dot(fvec, fvec) / 2
```

Comment:
I agree, remove the parenthesis. I just like explicit more than implicit. Don't care much though :)